item_db.js

```diff
@@ -1,6 +1,35 @@
-/*eslint no-undef: "error"*/
-/*eslint no-console: ["error", { allow: ["warn", "error"] }] */
 /*eslint-env node*/
 const alasql = require('alasql');
 
-module.exports = alasql('CREATE TABLE items (id int, name varchar(255), quantity int, unit varchar(25))');
+alasql('CREATE TABLE users (id INT PRIMARY KEY AUTOINCREMENT, username varchar(255), password varchar(255))');
```

Comment:
a good candidate for splitting into `/src/lib/setup-db.js` with the following api:
* setup()

and `/src/model/item.js` with api:
* findById() (or better getItemById)

and `/src/model/user.js` with api:
* findByUserName() (or better getUserByName())